scripts/run_test.py

```diff
@@ -280,25 +280,65 @@
     memservice_config_doc["ATL_queue_size"] = args.atlqsize
 if args.atldatasize is not None:
     memservice_config_doc["ATL_data_size"] = args.atldatasize
+'''
+In config file the memory server details are stored as given below (as map of map): 
+
+Memservers:
+ 0:
+   memory_type: volatile
+   fam_path: /dev/shm/vol/
+   rpc_interface: 127.0.0.1:8789
+   libfabric_port: 7500
+   if_device: eth0 
+
+ 1:
+   memory_type: persistent
+   fam_path: /dev/shm/per/
+   rpc_interface: 127.0.0.1:8788
+   libfabric_port: 7501
+   if_device: eth1 
+
+The input to it is passed to this script as :
```

Comment:
Please reword this comment line, it does not sound clear.
"The input to this script is passed as :"